plugins/polio/js/src/components/Inputs/OrgUnitsSelect.tsx

```diff
@@ -45,6 +46,7 @@ export const OrgUnitsLevels: FunctionComponent<Props> = ({
                 initialSelection={initialOrgUnit}
                 showStatusIconInTree={false}
                 showStatusIconInPicker={false}
+                // @ts-ignore
                 errors={errors}
```

Comment:
You can remove the `@ts-ignore` by casting the type of `errors`

```javascript
errors = {errors as string[]}
```

It's still not perfect but it makes more obvious that we expect an array of strings there